### PR TITLE
fix map update issue with old kernels

### PR DESCRIPTION
**Pull request by yanivagman**

### CODE REVIEW

**Comment by itaysk** [comment]
can you explain what this does please?

**Comment by yanivagman** [comment]
We found that with recent changes we added today to the send_bin function, Tracee doesn't run on kernel 4.15 (On my 4.19 there is no problem). This fixes the problem for 4.15

File: tracee/event_monitor_ebpf.c [hunk 1]
```diff
@@ -1724,8 +1724,13 @@ int send_bin(struct pt_regs *ctx)
     chunk_size = bin_args->full_size - i*F_CHUNK_SIZE;
 
     if (chunk_size > F_CHUNK_SIZE) {
-        bin_args->full_size = chunk_size;
-        bin_args_map.update(&id, bin_args);
+        bin_args_t bin_args_new = {};
```
Comment (itaysk) [inline tracee/event_monitor_ebpf.c:1727]:
```suggestion
        // create a copy of the map before submitting - workaround for an issue with old kernels
        bin_args_t bin_args_new = {};
```

Comment (yanivagman) [reply tracee/event_monitor_ebpf.c:1727]: It actually the same as it was before a previous commit - nothing odd here

**Pull request merged**